File: jena-arq/testing/ARQ/Scripting/test-library.py
barx = '||'
def bar(x, y):
    return barx + combine(x, y) + barx
def value(n):
    return n.value
def combine(x, y):
    try:
        return x + y
    except:
        return str(x) + str(y)
def identity(x):
    return x
def rtnBoolean():
    return True
def rtnString():
    return "foo"
def rtnInteger():
    return 57
def rtnDouble():
    return 5.7
def rtnUndef():
    return None
def rtnNull():
    return None

File: jena-arq/testing/ARQ/Scripting/test-library.js
var barx = '||';
function bar(x,y)       { return barx+combine(x,y)+barx } 
function value(n)       { return n.value ; }
function combine(x,y)   { return x+y }
function identity(x)    { return x }

// The JavaScript types (numbers split).
function rtnBoolean()   { return true; }
function rtnString()    { return "foo"; }
function rtnInteger()   { return 57 }
function rtnDouble()    { return 5.7 }
function rtnUndef()     { return undefined }
function rtnNull()      { return null; }
//function rtnSymbol()    { return Symbol('foo') }
